modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -151,33 +151,6 @@ output application/json
 
 === Example
 
-This example sorts an array of DateTime in descending order.
-
-==== Source
-
-[source,DataWeave,linenums]
-----
-%dw 2.0
-output application/json
----
-[|2020-10-01T23:57:59.017Z|, |2022-12-22T12:12:12.011Z|, |2020-10-01T12:40:10.012Z|, |2020-10-01T23:57:59.021Z|]
-  orderBy -($ as Number {unit: "milliseconds"})
-----
-
-==== Output
-
-[source,JSON,linenums]
-----
-[
-  "2022-12-22T12:12:12.011Z",
-  "2020-10-01T23:57:59.021Z",
-  "2020-10-01T23:57:59.017Z",
-  "2020-10-01T12:40:10.012Z"
-]
-----
-
-=== Example
-
```

Comment:
Do you think this example needs to be added to the data-weave repo? Or was it deleted intentionally?

Reply:
I didn't find references in the data-weave repo about this example, maybe it was added in docs repository only. You can add it to the data-weave repo.